### DISCUSSION TITLE
Are known senders automatically whitelisted?

### DISCUSSION THREAD

**Discussion by logkirk** [afterxleep/Bouncer#80] [Q&A] [answered]
This seems to be the case--I messaged myself using an alternate phone number (which already had message history and is in my contacts), with the message containing a word on my blocklist. The message came through fine.

If known senders are automatically whitelisted, I think this should be made clear in the documentation (did I miss it somewhere?) or there should be a default whitelist rule already populated for this.

**Comment by afterxleep** [comment, marked as answer]
Yes.  Apple automatically whitelists known senders.   This means, anyone you've interacted with before, or is in your contacts.
